### PR TITLE
Foundations for E.O. integration tests

### PR DESCRIPTION
**Pull request by KtorZ**
<p align="right">#236</p>

# Overview

- [x] I have added the necessary foundation for testing creation of externally owned wallet
    - This includes a new section in the fixture to generate a list of account public keys and their corresponding indexes from an initial backup phrase
    - Also includes the necessary lenses to extend the integration tests DSL

# Comments

### CODE REVIEW

File: src/Cardano/Wallet/API/V1/Types.hs [hunk 1]
```diff
@@ -1111,6 +1113,12 @@ instance Bounded AccountIndex where
     minBound = AccountIndex 2147483648
     maxBound = AccountIndex maxBound
 
+instance Enum AccountIndex where
+    fromEnum (AccountIndex a) = fromEnum a
+    toEnum a = case mkAccountIndex (toEnum a) of
```
Comment (uroboros) [inline src/Cardano/Wallet/API/V1/Types.hs:1118]: I can't quite make sense of that 2nd toEnum?

**Review by uroboros — APPROVED** [review]
just one question - otherwise LGTM

File: src/Cardano/Wallet/API/V1/Types.hs (hunk 1 above)
Comment (KtorZ) [reply src/Cardano/Wallet/API/V1/Types.hs:1118]: We have this `toEnum :: Int -> a` and `mkAccountIndex :: Word32 -> Either ... AccountIndex`, so the 2nd `toEnum` here is to go from `Int` to `Word32` :)

Comment (uroboros) [reply src/Cardano/Wallet/API/V1/Types.hs:1118]: ahhh thanks ;)

**Pull request merged**